resolwe_bio/tests/processes/test_generate_report.py

```diff
@@ -0,0 +1,42 @@
+# pylint: disable=missing-docstring
+from resolwe_bio.utils.test import skipDockerFailure, BioProcessTestCase
+
+
+class ReportProcessorTestCase(BioProcessTestCase):
+
+    @skipDockerFailure("Processor requires a custom Docker image.")
```

Comment:
I'll give you a more detailed review of the test in a separate issue, but I won't block the PR for it.